taskwiki/regexp.py

```diff
@@ -17,7 +17,7 @@
 DUE = r'(?P<due>{0})'.format(DUE_UNNAMED)
 TEXT = r'(?P<text>.+?)'
 COMPLETION_MARK = r'(?P<completed>.)'
-PRIORITY = r'(?P<priority>!{1,3})'
+PRIORITY = r'(?P<priority>!+|¡+)'
```

Comment:
I'm not a big fan of the reversed exclamation mark - while it really does look cool, it is not straightforward to type on the keyboard. I'm wondering if there is not a different symbol we could use that would naturally convey the meaning of lower priority.